x-pack/plugins/enterprise_search/common/types/index.ts

```diff
@@ -30,3 +30,14 @@ export interface IConfiguredLimits {
   appSearch: IAppSearchConfiguredLimits;
   workplaceSearch: IWorkplaceSearchConfiguredLimits;
 }
+
+export interface IMetaPage {
+  current: number;
+  size: number;
+  total_pages: number;
+  total_results: number;
+}
+
+export interface IMeta {
+  page: IMetaPage;
+}
```

Comment:
Nice, I think I remember @scottybollinger asking for this a while back. It looks like we're not using this anywhere in this PR though (since we removed hasValidData checks).

Just to tie this in with our other convo about splitting up PRs (apologies if I'm beating a dead horse here :), this would have been a great candidate as well for a separate & v granular PR. Scotty would have been able to use this type def sooner, & it helps clear up some minor confusion. I also find it's good practice getting used to Kibana - the more PRs you open, the easier their workflows (labels, backporting, running test and type checks, etc.) gets.